include/cantera/thermo/Phase.h

```diff
@@ -266,6 +266,19 @@ class Phase
     //! Return a const reference to the vector of species names
     const std::vector<std::string>& speciesNames() const;
 
+//    const std::vector<std::string>& speciesNamesNeutrals() const;
+
+    void speciesNamesPositiveIons(std::vector<std::string> & vec, std::vector<int> &vec2);
```

Comment:
These methods need to be documented (in particular, the meaning of the arguments. I would suggest more descriptive names than "vec" and "vec2").